src/Proxy/FilterPathProxy.php

```diff
@@ -17,7 +16,7 @@
 use ValueObjects\Web\Domain;
 use ValueObjects\Web\PortNumber;
 
-class CalendarSummaryProxy extends Proxy
+class FilterPathProxy extends Proxy
```

Comment:
Ik heb proxy uitgebreid met een extra `method` parameter en hernoemd naar `FilterPathMethodProxy`. In de Silex applicatie is er een bootstrap functie dat methode op `GET` zet voor de _calendar-summary_ en _search_ proxies